applications/luci-app-https_dns_proxy/luasrc/view/https_dns_proxy/buttons.htm

```diff
@@ -0,0 +1,56 @@
+<%#
+  Copyright 2019 Stan Grishin <stangri@melmac.net>
+-%>
+
+<%-
+	local packageName = "https_dns_proxy"
```

Comment:
Do we really need this packageName checks?
Could we not add **https_dns_proxy** as a dependency to the Makefile?

Reply:
I use similar code between different packages, it's easier for me to set the variable for the specific package when replace all entries in the code.

Reply:
I do not understand that!

Reply:
There's no dependency outside of the Makefile.

Reply:
That is the right source code place for this discussion https://github.com/openwrt/luci/pull/3215/files#r339785930